assets/src/containers/CourseList.js

```diff
@@ -60,7 +60,7 @@ function CourseList (props) {
       <AppBar className={classes.root} position='static'>
```

Comment:
This commit resolves the warning message
`tiny-warning.esm.js:13 Warning: You are attempting to use a basename on a page whose URL path does not begin with the basename. Expected path "/" to begin with "/courses".`